scanctl/cli.py

```diff
@@ -139,14 +141,22 @@ def ws_delete_projects(ctx, products):
               help='Token to use for Whitesource API access.')
 @click.option('--limit', type=click.INT, default=16,
               help='Maximum number of concurrent tasks while scanning.')
-def scan(repositories, file, remote, url, token, limit):
+@click.option('--fs-agent-jar', type=click.Path(exists=True),
+              default=os.path.join(os.getcwd(), 'fs-agent.jar'),
+              help='Path to Whitesource Filesystem Agent jar file.')
+@click.option('--fs-agent-config', type=click.Path(exists=True),
+              default=os.path.join(os.getcwd(), 'whitesource.config'),
+              help='Path to Whitesource Filesystem Agent config file.')
+def scan(repositories, file, remote, url, token, limit,
+         fs_agent_jar, fs_agent_config):
     if not (repositories or file):
         sys.exit('No repositories to scan.')
     if limit <= 0:
         sys.exit('Task limit cannot be less than 1.')
 
     sem = asyncio.Semaphore(limit)
-    agent = whitesource.FsAgent(token=token)
+    agent = whitesource.FsAgent(token=token,
+                                jar=fs_agent_jar, config=fs_agent_config)
```

Comment:
I'd prefer to see this [wrapped](https://github.com/ambv/black#how-black-wraps-lines) like so: 

```py
agent = whitesource.FsAgent(
    token=token, jar=fs_agent_jar, config=fs_agent_config,
)
```

It's a habit I like because git diffs are friendlier:

```diff
   agent = whitesource.FsAgent(
       token=token, jar=fs_agent_jar, config=fs_agent_config,
+      foo=arg1, bar=arg2, baz=arg3 ,
   )
```